contrib/completion/bash/docker-compose

```diff
@@ -93,7 +93,7 @@ __docker_compose_services_stopped() {
 _docker_compose_build() {
 	case "$cur" in
 		-*)
-			COMPREPLY=( $( compgen -W "--help --no-cache --pull" -- "$cur" ) )
+			COMPREPLY=( $( compgen -W "--help --force-rm --no-cache --pull" -- "$cur" ) )
```

Comment:
Please keep the options in alphabetical order here.
Otherwise, bash completion LGTM.
Thanks, Adrian!